dot/state/grandpa_test.go

```diff
@@ -143,7 +143,7 @@ func testBlockState(t *testing.T, db database.Database) *BlockState {
 	return bs
 }
 
-func TestAddScheduledChangesKeepTheRightForkTree(t *testing.T) {
+func TestAddScheduledChangesKeepTheRightForkTree(t *testing.T) { //nolint:tparallel
```

Comment:
I've introduced this `nolint` to avoid using `t.Parallel()` under `t.Run` test cases, this test specifically creates a chain with forks and uses the same chain for each use case